### PR TITLE
Statistics: a large number floating point test for cov

### PR DESCRIPTION
**Pull request by iblislin**

### CODE REVIEW

File: stdlib/Statistics/test/runtests.jl [hunk 1]
```diff
@@ -370,6 +370,12 @@ Y = [6.0  2.0;
         @test C ≈ Cxy
         @inferred cov(X, Y, dims=vd, corrected=cr)
     end
+
+    # case for large number, floating point accurcy issue
```
Comment (andreasnoack) [inline stdlib/Statistics/test/runtests.jl]: Would be great to make this comment the title of a `testset`. It helps when the test fails.

Comment (iblis17) [reply stdlib/Statistics/test/runtests.jl]: ah, okay

File: stdlib/Statistics/test/runtests.jl [hunk 2]
```diff
@@ -370,6 +370,12 @@ Y = [6.0  2.0;
         @test C ≈ Cxy
         @inferred cov(X, Y, dims=vd, corrected=cr)
     end
+
+    @testset "floating point accurcy in large number" begin
```
Comment (StefanKarpinski) [inline stdlib/Statistics/test/runtests.jl]:
```suggestion
    @testset "floating point accuracy `cov` of large numbers" begin
```

File: stdlib/Statistics/test/runtests.jl [hunk 3]
```diff
@@ -370,6 +370,12 @@ Y = [6.0  2.0;
         @test C ≈ Cxy
         @inferred cov(X, Y, dims=vd, corrected=cr)
     end
+
+    @testset "floating point accuracy for `cov` of large numbers" begin
+      A = [4.0, 7.0, 13.0, 16.0]
```
Comment (ararslan) [inline stdlib/Statistics/test/runtests.jl]: 4 space indents.

Comment (iblis17) [reply stdlib/Statistics/test/runtests.jl]:
```suggestion
        A = [4.0, 7.0, 13.0, 16.0]
```

Comment (iblis17) [reply stdlib/Statistics/test/runtests.jl]: I always forget that :man_facepalming:

**Review by ararslan — APPROVED** [review]

**Pull request merged**